_includes/build-instructions-short.md

```diff
@@ -225,6 +225,8 @@ gibt es die [detaillierte Aufbauanleitung](/hardware/v{{ include.version
 * GPS-Modul aufstecken 
 * Verlöten und Stifte kürzen
 
+**Achtung! Achte darauf, dass das kurze Ende der Stiftleiste in das Board und das lange Ende in das GPS-Modul eingelötet wird. Ansonsten ragt das GPS-Modul aus dem Gehäuse heraus.*
```

Comment:
Formatting is a bit weird. There should either be two `**` at the end, or one `*` at the beginning, then a space, then another `*`. I'd even do it like this: 

```
  * **Achtung!** Achte darauf ....
```

Such that only the "Achtung" is bold, not the whole line. That's easier to read IMO.

Reply:
Sorry, this was done on the go 👍 

I will fix this.